pallets/utxo/src/lib.rs

```diff
@@ -320,25 +394,24 @@ pub mod pallet {
             }
             ensure!(res.is_ok(), "header error. Please check the logs.");
 
-            match output.script.stype {
-                ScriptType::P2pkh => {
+            match output.destination {
+                Destination::Pubkey(_) => {
                     ensure!(output.value > 0, "output value must be nonzero");
                     let hash = BlakeTwo256::hash_of(&(&tx, output_index));
                     output_index = output_index.checked_add(1).ok_or("output index overflow")?;
                     ensure!(!<UtxoStore<T>>::contains_key(hash), "output already exists");
-
-                    // checked add bug in example cod where it uses checked_sub
-                    total_output =
-                        total_output.checked_add(output.value).ok_or("output value overflow")?;
                     new_utxos.push(hash.as_fixed_bytes().to_vec());
                 }
-                ScriptType::OpCreate => {
+                Destination::CreatePP(_, _) => {
                     log::info!("TODO validate OP_CREATE");
                 }
-                ScriptType::OpCall => {
+                Destination::CallPP(_, _) => {
                     log::info!("TODO validate OP_CALL");
                 }
             }
+            // checked add bug in example cod where it uses checked_sub
+            total_output =
```

Comment:
Extremely minor thing but running cargo-fmt told that this line is not formatted correctly. If you're doing any other changes to this commit anymore, the code could be formatted.

Reply:
Cool, will change that, thanks for pointing this out